financial_calculations/bond_cash_flows.py

```diff
@@ -54,13 +54,13 @@ def create_semi_bond_cash_flows(effective_date, balance, coupon, maturity_years)
         raise ValueError("Day of the month should not be greater than 28 to avoid end-of-month issues.")
 
     # Calculate the number of payments (semiannual payments)
-    num_payments = int(FRAC_OF_YEAR * maturity_years)
+    num_payments = int(PMTS_PER_YEAR * maturity_years)
 
     # Generate the payment dates by adding multiples of 6-month periods and change type to datetime64[D] object
     payment_dates = [np.datetime64(effective_date + relativedelta(months=6 * i), 'D') for i in range(1, num_payments + 1)]
```

Comment:
need to eventually get this into its own function.  don't want to duplicate functionality